drivers/flash/soc_flash_mcux.c

```diff
@@ -86,6 +86,21 @@ static int flash_mcux_write_protection(struct device *dev, bool enable)
 	return -EIO;
 }
 
+#if defined(CONFIG_FLASH_PAGE_LAYOUT)
+static const struct flash_pages_layout dev_layout = {
+	.pages_count = KB(CONFIG_FLASH_SIZE) / FLASH_ERASE_BLOCK_SIZE,
+	.pages_size = FLASH_ERASE_BLOCK_SIZE,
+};
+
+static void flash_mcux_pages_layout(struct device *dev,
+									const struct flash_pages_layout **layout,
```

Comment:
Are these misaligned or is that a github illusion ?

Reply:
github illusion